apps/src/templates/manageStudents/ManageStudentsActionsCell.jsx

```diff
@@ -33,7 +33,7 @@ const styles = {
   }
 };
 
-class ManageStudentActionsCell extends Component {
```

Comment:
this change is unrelated to the PR task, but included as a general QOL improvement. when i went to track the redux flow that was resetting our pilot flag, i had trouble finding usages of this component due to a typo in its name.